### ISSUE TITLE
TurnServer.java

### ISSUE THREAD

**Issue by Vivektilva** [vivek1794/webrtc-android-codelab#20]
in this interface , req url is there and for that i signed up in xirsys to get new values of param from my account.

from Xirsys account i got : ident,secret,channel,secure.
 
what should i pass in : domain,application,room??

**Comment by vickyzare567** [comment]
Hello vivek 
Do you solved the issue yet .. 
I am also stuck on the same problem

**Comment by Vivektilva** [comment]
ya instead of using API call , use static server detail.

**Comment by vickyzare567** [comment]
Can you please explain in some details 
 Static server means which server ..
On that we are using socket implementation  ?

**Comment by Vivektilva** [comment]
when you register your acc for stun and turn server( on xirsys) they will provide detail of API and also of static server detail .

**Comment by vickyzare567** [comment]
I got the details of stun and turn but can please tell me how to use it in TurnServer.java interface.   I have putted the details of api in this .

**Comment by Vivektilva** [comment]
no need to use Turnserver.java if u want to use static server detail, in xirsys where you are getting api detail in same page you will get static server detail, so instead of making API request  to get list ,use those static server and put them in list .

**Comment by vickyzare567** [comment]
Ok Thanks Vivek. I got it 
I just need to add the static servers to the list iceServers. 
Am I Right.. ?

**Comment by vickyzare567** [comment]
But I have one Confusion. Should i need to put both the url's of stun and Turn or only turn ..?

**Comment by Vivektilva** [comment]
yes right just need to add static server to the list iceServers , actually it will work with Turn server but in list put url of both 1st stun server and after that turn servers.
